packages/components/organisms/f-checkout/src/components/Error.vue

```diff
@@ -69,5 +123,9 @@ export default {
         @include font-size(body-l);
         margin-top: spacing();
     }
+
+    .c-checkout-error-button {
```

Comment:
Small one, but do `.c-checkout-error-heading`, `.c-checkout-error-description` and `.c-checkout-error-button` need to be nested?

If not, I'd un-nest them to keep the specificity of the CSS lower.